spiel/cmd.c

```diff
@@ -2194,13 +2194,29 @@ static int spiel_process__counters_async(struct spiel_proc_counter_item *proc)
 				  &proc->sci_rlink_wait);
 	return M0_RC(rc);
 }
+void m0_spiel_count_stats_init(struct m0_proc_counter *count_stats)
+{
+	return;
+}
+M0_EXPORTED(m0_spiel_count_stats_init);
+
+void m0_spiel_count_stats_fini(struct m0_proc_counter *count_stats)
+{
+	if(count_stats->pc_cnt == 0)
+		return;
+
+	if(count_stats != NULL)
+		m0_spiel_count_stats_free(count_stats,
```

Comment:
Here, elements of the array pc_bckey and pc_bcrec from index 0 to count_stats->pc_cnt are getting freed, but the array itself is still not freed. Where are we planning to free them?